From ef45c7ce5325d85ca924502b1b0a929fecd6581d Mon Sep 17 00:00:00 2001
From: Stephan Hoffmann <sho@relinux.de>
Date: Thu, 27 Dec 2012 18:08:29 +0100
Subject: [PATCH] Fix test for presence of X11

The test in GNUmakefile.in checking for presence of X11 and
enabling the XP_UNIX define succeeds even when X11 is not
present and Webkit is built for DirectFB.

Because of this X11-headers are included and the build fails.

Signed-off-by: Stephan Hoffmann <sho@relinux.de>
---
 GNUmakefile.in |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/GNUmakefile.in b/GNUmakefile.in
index f14f27b..3ac811f 100644
--- a/GNUmakefile.in
+++ b/GNUmakefile.in
@@ -88,7 +88,8 @@ noinst_PROGRAMS = Programs/minidom$(EXEEXT) \
 	$(am__EXEEXT_1)
 
 # For the Gtk port we want to use XP_UNIX both in X11 and Mac
-@TARGET_WIN32_FALSE@am__append_1 = -DXP_UNIX
+@TARGET_X11_TRUE@am__append_1 = -DXP_UNIX
+@TARGET_QUARTZ_TRUE@am__append_1 = -DXP_UNIX
 @USE_ICU_UNICODE_TRUE@am__append_2 = \
 @USE_ICU_UNICODE_TRUE@	-DWTF_USE_ICU_UNICODE=1
 
-- 
1.7.0.4

